ivy/functional/frontends/numpy/linear_algebra/matrix_and_vector_products.py

```diff
@@ -0,0 +1,43 @@
+# matmul
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_np.valid_float_dtypes,
+        min_num_dims=3,
+        max_num_dims=5,
+        min_dim_size=2,
+        max_dim_size=5,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.linalg.matmul"
+    ),
+    fm=st.booleans(),

```

Comment:
this is incorrect, please check the example mentioned below

Reply:
Done